components/nexus-repository/src/main/java/org/sonatype/nexus/repository/negativecache/ValueNegativeCacheKey.java

```diff
@@ -14,6 +14,8 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+// TODO: implement Externalizable
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)